src/models/user.js

```diff
@@ -42,7 +42,44 @@ module.exports = (sequelize, DataTypes) => {
       facebook: {
         type: DataTypes.STRING,
         required: false
-      }
+      },
+      gender: {
+        type: DataTypes.STRING,
+        required: false
+      },
+      birthday: {
+        type: DataTypes.DATE,
+        required: false
+      },
+      preferredlanguage: {
+        type: DataTypes.STRING,
+        required: false
+      },
+      currency: {
+        type: DataTypes.STRING,
+        required: false
+      },
+      residentialaddress: {
+        type: DataTypes.STRING,
+        required: false
+      },
+      role: {
+        type: DataTypes.ENUM,
+        values: [
+          'Developer',
+          'QA',
+          'Maintenance',
+        ],
+        required: false
+      },
+      department: {
+        type: DataTypes.STRING,
+        required: false
+      },
+      linemanager: {
+        type: DataTypes.INTEGER,
```

Comment:
Please take a look at your line manager's data type to allow for referencing to a data type corresponding to UUID which saves the userId.